### PR TITLE
feat: implement sessionOwner - better 'used elsewhere' ux

### PR DESCRIPTION
**Pull request by mroz22**
main changes:

- replace text "device detected in incorrect state" with "device used elsewhere". There is nothing wrong with the device. it is just used elsewhere.
- make information about what is elsewhere in the UI.

before:

image

afrer: 

image

image

related:
- #5543
- depend on #14776
- depends on #15080

### CODE REVIEW

File: packages/transport-bridge/src/core.ts [hunk 1]
```diff
@@ -120,11 +120,12 @@ export const createCore = (apiArg: 'usb' | 'udp' | AbstractApi, logger?: Log) =>
         acquireInput: Omit<AcquireInput, 'previous'> & {
             previous: Session | 'null';
             signal: AbortSignal;
-        },
+        } & { sessionOwner: string },
     ) => {
         const acquireIntentResult = await sessionsClient.acquireIntent({
             path: acquireInput.path,
             previous: acquireInput.previous === 'null' ? null : acquireInput.previous,
+            sessionOwner: acquireInput.sessionOwner,
```
Comment (marekrjpolak) [inline packages/transport-bridge/src/core.ts]: NOTE: is it necessary to have `sessionOwner` in both `acquireIntent` and `acquireDone`?

File: packages/transport-bridge/tests/http.test.ts [hunk 2]
```diff
@@ -562,13 +602,11 @@ describe('http', () => {
             await bridgeApiCall({
                 url: url + 'enumerate',
                 method: 'POST',
-                body: {},
```
Comment (marekrjpolak) [inline packages/transport-bridge/tests/http.test.ts:565]: NOTE: why?

File: packages/transport-bridge/src/core.ts (hunk 1 above)
Comment (mroz22) [reply packages/transport-bridge/src/core.ts]: nope, not needed. was needed before your refactoring, not anymore

File: packages/transport-bridge/tests/http.test.ts (hunk 2 above)
Comment (mroz22) [reply packages/transport-bridge/tests/http.test.ts:565]: upsie, put back

**Review by marekrjpolak — APPROVED** [review]

**Pull request merged**